infoscreen/models/tag.py

```diff
@@ -9,27 +9,28 @@
 
 # End: imports -----------------------------------------------------------------
 
-        
+
 class Tag(root_models.CustomBaseModel):
-    name = models.CharField(max_length=200, unique=True, null=False, blank=False, verbose_name='navn', help_text="En vilkårlig egenskap til en plante. (Tips: Du kan prefikse tags med kolon ':', f.eks. 'familie:fiola' )")
+    name = models.CharField(
+        max_length=200, unique=True, null=False, blank=False, verbose_name='navn', help_text='En vilkårlig egenskap til en plante. (Tips: Du kan prefikse tags med kolon ":", f.eks. "familie:fiola" )'
```

Comment:
Kan ta bort help_text her.
Avslutt også en lang liste med komma for bedre yapf formatering.

Reply:
Fikset